File: testdata/jennies/rawtypes/struct_with_optional_fields/PHPRawTypes/src/Types/StructOptionalFields/SomeOtherStruct.php
<?php

namespace Grafana\Foundation\Types\StructOptionalFields;

class SomeOtherStruct implements \JsonSerializable {
    /**
     * @var mixed
     */
    public $fieldAny;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "FieldAny" => $this->fieldAny,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/struct_with_optional_fields/CSharpRawTypes/src/Grafana/Foundation/StructOptionalFields/SomeOtherStruct.cs
namespace Grafana.Foundation.StructOptionalFields;


public class SomeOtherStruct
{
    public object FieldAny;

    public SomeOtherStruct()
    {
        this.FieldAny = new object();
    }

    public SomeOtherStruct(object fieldAny)
    {
        this.FieldAny = fieldAny;
    }
}
